app/bundles/LeadBundle/Form/DataTransformer/TagEntityModelTransformer.php

```diff
@@ -38,10 +35,9 @@ class TagEntityModelTransformer implements DataTransformerInterface
 
     /**
      * @param string $repo
-     * @param string $identifier
      * @param bool   $isArray
      */
-    public function __construct(EntityManager $em, $repo = '', $identifier = 'id', $isArray = false)
+    public function __construct(EntityManager $em, $repo = '', $isArray = false)
```

Comment:
Inconsistent phpdoc

Reply:
Please be specific

Reply:
EntityManager is missing. If it is new accepted coding standard I need to accept it and somehow fix PHPStorm code inspections. Never seen this before